backend/migrations/20201009195112_nativeLanguages.js

```diff
@@ -0,0 +1,13 @@
+const { tableNativeLanguages } = require('../src/models/nativeLanguage')
+
+exports.up = (knex) => {
+  return knex.schema.createTable(tableNativeLanguages, (table) => {
+    table.increments('NativeLanguageID').primary() // ToDo StatusID vs ID (postgres issue). Figure out uuid
```

Comment:
Remove this TODO comment.
```suggestion
    table.increments('NativeLanguageID').primary()
```